Automattic/woocommerce-payments
# Transactions list: Add conversion icon with tooltip showing original amount

Fixes #1202

#### Changes proposed in this Pull Request





Adds an icon next to the amount in the Transactions list row when the amount is converted from a different customer currency. A tooltip shows up on hover, with the amount in the original currency shown.





<img width="291" src="https://user-images.githubusercontent.com/1867547/107321725-0d2f8980-6a71-11eb-943a-e51dee6d6a21.png">
…and on hover: <img width="291" src="https://user-images.githubusercontent.com/1867547/107321723-0c96f300-6a71-11eb-9f6c-4ab83721559e.png">

Remaining known issues here:
- The tooltip **doesn't show up on iPhone** (not tested on any other touchscreen devices) – not sure why but I suspect it's not specific to this usage of the <Tooltip> component. (I'd originally removed clickable from this cell, hoping it would make the Tooltip work.) Ideas? FYI @LevinMedia
- <img width="261" src="https://user-images.githubusercontent.com/1867547/107326431-67cce380-6a79-11eb-91b0-25c8404a1ed6.png"> – see https://github.com/Automattic/woocommerce-payments/issues/1301

#### Testing instructions





Make orders with WCPay using the store currency and various other currencies (e.g. using the WooCommerce Multi-currency extension). Verify that the amounts appear in the transactions list:
- with no icon when the order currency matches the settlement currency on the Stripe account
- with an icon when it doesn't, showing the original amount on hover

-------------------
